snappy/info.go

```diff
@@ -184,7 +188,41 @@ func PackageNameActive(name string) bool {
 	return ActiveSnapByName(name) != nil
 }
 
-// RemoteManifestPath returns the would be path for the store manifest meta data
-func RemoteManifestPath(s *snap.Info) string {
+// ManifestPath returns the would be path for the snap manifest.
+func ManifestPath(s *snap.Info) string {
 	return filepath.Join(dirs.SnapMetaDir, fmt.Sprintf("%s_%s.manifest", s.Name, s.Version))
 }
+
+/// XXX: temporary step until we know if we still need this (vs overlord state)
+// and to find out what we absolutely need from it
+type diskManifest struct {
+	// XXX likely we want also name and snap-id and summary
+	Revision    int    `yaml:"revision"`
+	Channel     string `yaml:"channel"`
+	Developer   string `yaml:"developer"`
+	Description string `yaml:"description"`
+	Size        int64  `yaml:"size"`
```

Comment:
@mvo5 I think you are right about iconurl, will add it and add some code to snap_local.go for now, will refactor in later branches as needed